modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -8,10 +8,27 @@
       <AposMinMaxCount
         :field="field"
         :value="next"
+        v-if="!field.inline"
```

Comment:
Is this another thing to realize the design? Is the min and max information apparent in another way?

Reply:
yes but this is on the wrong PR, I'm moving it